cloud/openstack/os_user.py

```diff
@@ -154,7 +154,8 @@ def main():
         module.fail_json(msg='shade is required for this module')
 
     name = module.params['name']
-    password = module.params['password']
+    # pop the password so as not to impact the shade constructor
+    password = module.pop('password')
```

Comment:
Yes that worked for me (Ansible == 2.1.2.0 and passing auth via environment).  Thanks.